pages/download.py

```diff
@@ -188,13 +182,26 @@ def get_content(self):
 installation time. If you try to run two instances with the same identity _at
 the same time_, both proxy demons will become confused and upset. Don't do
 this!
+""")))+run_show_hide_script())
 
+class UsingSection(Section):
+    def __init__(self):
+        self.title = _("Using Freenet")
+        self.slug = "usage"
+    def get_content(self):
+        # License for all content in this section: GFDL (from old freenetproject.org website)
+        return text(md(_("""
+[Step by step guide](http://freesocial.draketo.de/) to setting up Freenet and
+various Freenet apps. Please try this, especially if installing on OS X. We
+are not responsible for unofficial third party apps it recommends (including
+FMS), but many Freenet users and developers use them.
+""") + _("""
 ### HOWTO
```

Comment:
While we're at it, there's no such word as "HOWTO". It's spelled "how-to", but even that doesn't remotely seem to capture the subject of this paragraph ;-)